files/fr/web/performance/how_long_is_too_long/index.html

```diff
@@ -0,0 +1,28 @@
+translation_of: 'Web/Performance/How_long_is_too_long'
+---
+<p>Il n'y a pas de règle stricte définissant ce qui constitue un site trop long à charger, mais il y a des bonnes pratiques spécifiques pour définir un bon temps de chargement du contenu (1&nbsp;seconde), de fonctionnement au ralenti (50&nbsp;millisecondes), d'animation ( 16,7&nbsp;secondes) ou encore de réponse à la saisie d'un visiteur (50 à 2000&nbsp;millisecondes).</p>
+
+<h3 id="load_goal">Objectifs de chargement</h3>
+
+<p>La durée optimale de temps de chargement est souvent définie comme étant «&nbsp;inférieure à une seconde&nbsp;», mais qu'est-ce que cela signifie&nbsp;? Ce temps d'une seconde devrait être considéré comme le temps maximal pour indiquer à une personne visitant le site que la demande à été traitée et que le nouveau contenu a été chargé, ce qui signifie par exemple que le navigateur affiche le titre de la page et sa couleur de fond.</p>
+
+<p>La première ressource récupérée après une demande est généralement un document HTML, qui appelle lui-même par la suite des ressources additionnelles. Comme noté dans la description du <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu (en anglais <i lang="en">critical rendering path</i>)</a>, lorsque la ressource est reçue alors les navigateurs commencent à traiter le HTML et affichent son rendu au fur et à mesure qu'il est reçu plutôt que d'attendre le chargement des ressources additionnelles.</p>
+
+<p>Oui, un temps de chargement d'une seconde est un bon objectif, mais c'est un objectif qui ne sera rempli que par quelques sites seulement. Les attentes diffèrent en réalité en fonction du contexte. L'affichage du texte «&nbsp;Bonjour tout le monde&nbsp;» devrait pouvoir s'afficher en quelques millisecondes sur un réseau d'entreprise, mais le téléchargement de la vidéo d'un chat rigolo sur un appareil de plus de 5 ans avec un réseau <i lang="en">Edge</i> dans le Nord de la Sibérie pourrait prendre plus de 20&nbsp;secondes sans choquer personne. D'une manière plus générale, si vous attendez trois ou quatre secondes avant de fournir le moindre contenu à la personne visitant votre site, vous perdrez des visiteurs potentiels qui ne reviendront peut-être jamais consulter votre site.</p>
+
+<p>En optimisant les performances de votre site, visez un temps de premier chargement du contenu ambitieux, comme par exemple 5 secondes sur un réseau mobile 3G, et 1,5 secondes sur un réseau Wifi, avec peut-être des objectifs de chargement de page encore plus ambitieux pour les chargements de page ultérieurs, en tirant parti des <i lang="en">service workers</i> et de la mise en cache. Il y a plusieurs moments différents pour initialiser le chargement de la page, le chargement des ressources additionnelles, la réponse à l'interaction de l'utilisateur ou de l'utilisatrice, et pour proposer des animations fluides :</p>
+
+<h3 id="idling_goal">Objectifs concernant le statut (en anglais <i lang="en">idling</i>)</h3>
```

Comment:
disclaimer: I don't know the French jargon around this, please let me know if there is any coined term :x 

From what I gather, the "idling goal" revolves around letting enough "time" for the page to react to user interactions. To me, the term "statut" doesn't convey this. That being written, this one is a bit hard if we want to avoid a looooong title.
I think "Objectifs concernant la réactivité générale" ou "Objectifs concernant le temps libre laissé à la page" (a bit long) would work though I'm not 100% decisive.

Reply:
Oh, and now that I've read the whole article, I realize "Réactivité" would fit perfectly for the last section :x

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1854/commits/8a5dbb598d0e4d7f8ab9cb68ca6db216ea9eafb8
New title: "Objectifs concernant les étapes de chargement de la page"